### ISSUE TITLE
Fused 1D modules for quantization

### ISSUE THREAD

**Issue by snakers4** [pytorch/pytorch#32768]
## 🚀 Feature

Fused 1D modules for [quantization](https://pytorch.org/docs/stable/quantization.html).

## Motivation

1D (separable) convolutions are building blocks for sound networks / time series / sequential data.

## Pitch

Ideally, quantization should support 1D as well as 2D operations out-of-the-box.

## Alternatives

Using 2D convolutions.
Although there are approaches and even papers, I could not replicated the papers and / or make them work properly. 

Nevertheless I achieved success with 1D convolutions.
Also this makes sense for data which is 1D in its essense, i.e. time series.

cc @jerryzh168 @jianyuh @dzhulgakov @raghuramank100 @jamesr66a

**Comment by raghuramank100** [comment]
@snakers4 : Can you elaborate on what fusions you are looking for? Is this conv1D->relu?

**Comment by snakers4** [comment]
@raghuramank100 

> Can you elaborate on what fusions you are looking for? Is this conv1D->relu?

I use mostly Conv1D => BatchNorm1D=> relu

```
modules.extend([conv_block(**conv_kwargs)])  # Conv1D
modules.extend([norm_block(out_ch, momentum=bnm), # bnorm
                            nonlinearity,  # relu
                            dropout])
```

**Comment by Haxxardoux** [comment]
Would love to see support for 1d operation fusion similar to the current 2d operation fusion, particularly the sequence mentioned above

**Comment by vkuzo** [comment]
this is supported in recent versions of PyTorch
